gcp/modules/gcp-stackdriver-monitoring/main.tf

```diff
@@ -37,6 +38,7 @@ resource "template_dir" "resources" {
     organization_id    = "${var.organization_id}"
     domain_name        = "${var.domain_name}"
     notification_email = "${(var.use_auth_user_email && var.auth_user_email != "") ? var.auth_user_email : var.notification_email}"
+    enabled_in_prd     = "${(var.env == "prd") ? true : false}"
```

Comment:
I would consider renaming this variable to generic `enabled`, or maybe smth. like `backup_alerst_enabled` as it seems to be only used by backup alerts. Then it would work if we decide to enable the alerts in other envs.